src/Microsoft.AspNetCore.OData/Results/BadRequestODataResult.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.OData;
+
+namespace Microsoft.AspNet.OData.Results
+{
+    /// <summary>
+    /// Represents a result that when executed will produce a Bad Request (400) response.
+    /// </summary>
+    /// <remarks>This result creates an <see cref="ODataError"/> with status code: 400.</remarks>
+    public class BadRequestODataResult : BadRequestResult
+    {
+        private string _message;
+
+        /// <summary>
+        /// Instantiate the Class.
```

Comment:
>Instantiate the Class. [](start = 12, length = 22)

Initializes a new instance of the <see cref="BadRequestODataResult" /> class.

Reply:
For others

---
In reply to: [534677066](https://github.com/OData/WebApi/pull/2341#discussion_r534677066) [](ancestors = 534677066)

Reply:
#Resolved